layouts/partials/nav.html

```diff
@@ -1,7 +1,7 @@
 {{- $s := .Site }}
 {{- $sp := $s.Params }}
 <nav class="wrap nav menu">
-	<a href="{{ $s.BaseURL }}" class="nav_brand">
+	<a href='{{ relLangURL "" }}' class="nav_brand">
```

Comment:
@ltGuillaume, can we use [absLangURL](https://gohugo.io/functions/abslangurl/)? Relative urls usually breakdown when a site is hosted within a subdirectory

Reply:
Should be ok.